tee-worker/ts-tests/indirect_calls.ts

```diff
@@ -59,14 +57,18 @@ export async function createIdentity(
     const encode = context.substrate.createType('LitentryIdentity', identity).toHex();
     const ciphertext = encryptWithTeeShieldingKey(context.teeShieldingKey, encode).toString('hex');
 
-    const tx =context.substrate.tx.identityManagement
-        .createIdentity(context.shard, signer.address, `0x${ciphertext}`, null)
-    
+    const tx = context.substrate.tx.identityManagement.createIdentity(
+        context.shard,
+        signer.address,
+        `0x${ciphertext}`,
+        null
+    );
```

Comment:
These format changes are strange -- did it follow `.prettierrc` or use other formatters?

Reply:
I use the format configured locally.